client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/site-setup-design-picker.tsx

```diff
@@ -295,6 +302,7 @@ const SiteSetupDesignPicker: Step = ( { navigation, flow } ) => {
 		recordTracksEvent( 'calypso_signup_design_view_more_select' );
 
 		setSelectedDesign( undefined );
+		setStepData( showGeneratedDesigns );
```

Comment:
It's weird to override the step data to a boolean value 😅

Reply:
It's redundant, I am about to remove it